### PR TITLE
feereel changes

### PR DESCRIPTION
**Pull request by feereel**
afd
adsdfasdfsadfasdf

### CODE REVIEW

File: testing_docker/.idea/.gitignore [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
```
Comment (stancej) [inline testing_docker/.idea/.gitignore:2]: sdafasdf

File: testing_docker/.idea/.gitignore [hunk 2]
```diff
@@ -1,8 +0,0 @@
...
-/workspace.xml
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
```
Comment (stancej) [inline testing_docker/.idea/.gitignore:6]: what this

**Review by stancej — CHANGES_REQUESTED** [review]

File: testing_docker/.idea/.gitignore (hunk 1 above)
Comment (feereel) [reply testing_docker/.idea/.gitignore:2]: good soluthion!!!!

File: testing_docker/.idea/.gitignore (hunk 2 above)
Comment (feereel) [reply testing_docker/.idea/.gitignore:6]: and this too

**Comment by feereel** [comment]
all work fine

**Comment by feereel** [comment]
dsfsadf

**Comment by stancej** [comment]
```C#

public void(int a, int b) => a+b;

```

**Comment by stancej** [comment]
> ```cs
> public void(int a, int b) => a+b;
> ```

Delete this

**Comment by feereel** [comment]
Bufo_bufo_03-clean

**Pull request merged**